pkg/crypto/pedersen/pedersen.go

```diff
@@ -8,6 +8,39 @@ import (
 	"math/big"
 )
 
+// divMod finds a nonnegative integer x < p such that (m * x) % p == n.
+// Assumes that m and p are coprime. This implementation is only meant
+// to be used in `pedersen.add`, where this assumption holds.
+// See https://github.com/starkware-libs/cairo-lang/blob/2abd303e1808612b724bc1412b2b5babd04bb4e7/src/starkware/crypto/starkware/crypto/signature/math_utils.py#L50-L56
+func divMod(n, m, p *big.Int) *big.Int {
+	a := new(big.Int)
+	new(big.Int).GCD(a, new(big.Int), m, p)
+	r := new(big.Int).Mul(n, a)
+	return r.Mod(r, p)
+}
+
+// add returns the sum of (x1, y1) and (x2, y2). It assumes that
```

Comment:
Note that this is not a plain addition to two tuples but rather an elliptic curve addition in your documentation.